Extensions/TileMap/helper/TileMapHelper.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=TileMapHelper.js.map
```

Comment:
should we add a comment to say where this file is coming from? This could be helpful in the future if we want to update it

Reply:
I added a readme:
* https://github.com/D8H/GDevelop/tree/TileMapHitBoxes/Extensions/TileMap/helper